src/OpenPr0gramm/Service/IPr0grammUserService.cs

```diff
@@ -41,6 +41,9 @@ public interface IPr0grammUserService
         [Post("/user/logout")]
         Task<Pr0grammResponse> LogOut([Body(BodySerializationMethod.UrlEncoded)]LogOutData data);
 
+        [Post("/user/captcha")]
+        Task<CaptchaResponse> Captcha();
```

Comment:
I'd call this ReuqestCaptcha or something similar.

Reply:
Done.